plugins/github/server/processors/IntegrationDeletedProcessor.ts

```diff
@@ -0,0 +1,64 @@
+import { App, Octokit } from "octokit";
+import { IntegrationService, IntegrationType } from "@shared/types";
+import env from "@server/env";
+import { Integration } from "@server/models";
+import BaseProcessor from "@server/queues/processors/BaseProcessor";
+import { IntegrationEvent, Event } from "@server/types";
+
+export default class IntegrationDeletedProcessor extends BaseProcessor {
+  static applicableEvents: Event["name"][] = ["integrations.delete"];
+
+  private githubClient?: Octokit;
+
+  constructor() {
+    super();
+    if (!env.GITHUB_APP_ID || !env.GITHUB_APP_PRIVATE_KEY) {
```

Comment:
This is going to prevent the deletion logic running entirely for eg Slack when the GitHub integration is not enabled

Reply:
The deletion seems to be working fine for Slack when I comment out `GITHUB_APP_ID` and `GITHUB_APP_PRIVATE_KEY`. What am I missing?

Reply:
You're right, sorry this was pre-coffee.